app/code/Transoft/Blog/Model/ModelRepository.php

```diff
@@ -0,0 +1,207 @@
+<?php
```

Comment:
Why it is called **ModelRepository**? Are you creating a **model** or a **blog**?

Reply:
fixed